extensions/amp-playbuzz/0.1/amp-playbuzz.js

```diff
@@ -181,8 +184,9 @@ class AmpPlaybuzz extends AMP.BaseElement {
       this.iframeLoaded_ = true;
       this.attemptChangeHeight(this.itemHeight_).catch(() => {/* die */ });
 
-      const unlisten = this.getViewport().onChanged(
+      const unlisten = events.listen(this.win,'scroll',
```

Comment:
Yes. Please definitely give it a try. And please note, `window.onscroll` simply doesn't work in all cases. AMP takes care of this via `viewport.onScroll`.